src/components/views/right_panel/VerificationPanel.js

```diff
@@ -69,6 +70,33 @@ export default class VerificationPanel extends React.PureComponent {
         const {member} = this.props;
         const AccessibleButton = sdk.getComponent('elements.AccessibleButton');
 
+        if (this.props.layout === 'dialog') {
+            // HACK: This is a terrible idea.
```

Comment:
(Idly wondering whether a subsequent PR revises this, or if we're living with it...)

Reply:
I tried to refactor it but couldn't find a good way to get rid of the `layout` prop as it would be very tricky to to achieve the two layouts with the same DOM, and also the copy is different. I think this is still preferable to duplicating the `VerificationPanel` code, so I'd propose we live with it for now.

Reply:
https://github.com/vector-im/riot-web/issues/12319